tools/SourceKit/tools/sourcekitd/bin/XPC/Client/CMakeLists.txt

```diff
@@ -1,6 +1,14 @@
-set(public_headers
-  ${SOURCEKITD_SOURCE_DIR}/include/sourcekitd/sourcekitd.h
-)
+set(public_headers)
+if (XCODE)
+  # Xcode does not like that we use this file as a header but also include it in
+  # a framework. The warning suggests, we copy it to a different path and
```

Comment:
Please don't do this. This is a header people edit, and having two copies means it will get out of sync.